test/values-expression.test.js

```diff
@@ -3,7 +3,7 @@ import { resetTracking } from "../src/track";
 
 function getConvertedValue(metric, target){
     let value = getValue(metric, target)
-    if (!value) return undefined;
+    if (value === null || value === undefined) return undefined;
```

Comment:
@rcowin is there a reason for this update?  I was just testing in the console and the simpler version you had seemed to work the same.

Reply:
When value = 0, they behave differently. This issue became visible with the tests for the new feature.